reports/funcion_logistica.tex

```diff
@@ -44,14 +79,75 @@
 \end{equation}
-\end{document}
\ No newline at end of file
+
+
+\newpage
+%Agregar valores de .csv
+
+\begin{table}[ht]
+    \centering
+    \caption{Variables predictoras para principales componentes de análisis de ($n$=135) albatros de Laysan en Isla Guadalupe desde 2014-2018.}
+    \bigskip
+    \begin{adjustbox}{width=1\textwidth}
+    \renewcommand{\arraystretch}{1.3}
+    \begin{tabular}{|c|c|c|c|c|c|c|}
+    \hline
+    \textbf{Morfometría} & \textbf{Valor Mínimo} & \textbf{Valor Máximo} & \textbf{Valor Estimado} & \textbf{Error Estándar} & \textbf{Valor Z} & \textbf{Pr($>\left|z\right|$)} \\
+    \hline
+    (Intercept) & - & - & \py{m_variables_modelo[0]['(Intercept)']} & \py{m_variables_modelo[0]['stdErrIntercept']} & \py{m_variables_modelo[0]['zValueIntercept']} & \py{m_variables_modelo[0]['PrIntercept']}  \\
+    \hline
+    Longitud Craneo & \py{m_variables_modelo[0]['minlongitudCraneo']} & \py{m_variables_modelo[0]['maxlongitudCraneo']} & \py{m_variables_modelo[0]['Longitud_Craneo']} & \py{m_variables_modelo[0]['stdErrlongitudCraneo']} & \py{m_variables_modelo[0]['zValuelongitudCraneo']} & \py{m_variables_modelo[0]['PrIntercept']}\\
+    \hline
+    Altura Pico & \py{m_variables_modelo[0]['minAltoPico']} & \py{m_variables_modelo[0]['maxAltoPico']} & \py{m_variables_modelo[0]['Altura_Pico']} & \py{m_variables_modelo[0]['stdErrAltoPico']} & \py{m_variables_modelo[0]['zValueAltoPico']} & \py{m_variables_modelo[0]['PrAltoPico']}\\
+    \hline
+    Longitud Pico & \py{m_variables_modelo[0]['minLongitudPico']} & \py{m_variables_modelo[0]['maxLongitudPico']} & \py{m_variables_modelo[0]['Longitud_Pico']} & \py{m_variables_modelo[0]['stdErrLongitudPico']} & \py{m_variables_modelo[0]['zValueLongitudPico']} & \py{m_variables_modelo[0]['PrLongitudPico']}\\
+    \hline
+    Tarso & \py{m_variables_modelo[0]['minTarso']} & \py{m_variables_modelo[0]['maxTarso']} & \py{m_variables_modelo[0]['Tarso']} & \py{m_variables_modelo[0]['stdErrTarso']} & \py{m_variables_modelo[0]['zValueTarso']} & \py{m_variables_modelo[0]['PrTarso']}\\
+    \hline
+    Ancho Craneo & \py{m_variables_modelo[0]['minAnchoCraneo']} &\py{m_variables_modelo[0]['maxAnchoCraneo']} &\py{m_variables_modelo[0]['Ancho_Craneo']} &\py{m_variables_modelo[0]['stdErrAnchoCraneo']} &\py{m_variables_modelo[0]['zValueAnchoCraneo']} & \py{m_variables_modelo[0]['PrAnchoCraneo']} \\
+    \hline
+    \end{tabular}
+    \end{adjustbox}
+    \\[10pt]
+    Resultados obtenidos del archivo \texttt{tabla\_mejores\_modelos.csv}.
+    \label{modeloLogistico}
+\end{table}
+
+%Agrega valores de .json
+
+\begin{table}[h]
+    \centering
+    \caption{Variables predictoras obtenidas después de la normalizacion según la ecuación (\ref{normalizacion}) para principales componentes de análisis de ($n$=135) albatros de Laysan en Isla Guadalupe desde 2014-2018.}
```

Comment:
No estoy seguro de la información que hay en las tablas. Haciendo chinitos en los ojos para entender mejor, creo que la información es: el máximo (las be i de la ecuación cinco) y mínimo (las a i de la ecuación cuatro) y el valor estimado serán las betas

Reply:
Te parece si agregando algo de texto antes de las tablas, algo como: 
> Lo anterior nos permite generar el valor máximo de las variables a partir de la ecuación cinco y...

Reply:
Esto con la intensión de no tener que agregar en ambas tablas el mismo choro de dónde se obtuvieron los valores, que no se vea taaan repetitivo

Reply:
Creo que suena bien tu propuesta